sktime/utils/results_writing.py

```diff
@@ -80,6 +109,144 @@ def write_results_to_uea_format(path, strategy_name, dataset_name, y_true,
     file.close()
 
 
+def write_dataframe_to_tsfile(
+    data,
+    path,
```

Comment:
Does path not include the file name? Below it says that `problem_name` gives the file name. So does this give the directory?

Reply:
no, path just refers to the directory to store the file, while problem_name is used to name the ts file at the same time.

Reply:
Okay I think this is worth clarifying in the doc strings, usually input args are data and path including file name (e.g. `np.savetxt` or `pd.to_csv`)